app/views/users/index.html.erb

```diff
@@ -1,6 +1,9 @@
 <div class="main-container">
 <div class="sign-main">
   <%= button_to "Sign out", destroy_user_session_path, method: :delete, class: "devise-button" %>
+  <% if user_signed_in? %>
+    <p class="logged-as">Logged in as <%= current_user.try(:name) %></p>
+  <% end %>
   <% unless user_signed_in? %>
   <button class="devise-button">
     <%= link_to "Log in", new_user_session_path, class: "link-login"%>
```

Comment:
- [ ] It would be better if we are able to register and log in the application and test the cancancan functionality in action. However, I get an error below when i try to register or signup
![Screenshot from 2023-03-09 04-41-28](https://user-images.githubusercontent.com/46388113/223918456-811ccda9-5003-4641-aed8-2069bc8e9c9a.png)

Reply:
Hi @addod19 , thank you for the comment, we will fix that!